### PR TITLE
Adding new problem

### PR DESCRIPTION
**Pull request by tkazi0**
For assignment 3 of SF-1

### CODE REVIEW

File: preceding-number-25/description.md
Comment (Sandor-B) [inline preceding-number-25/description.md:1]: This file is good. No errors.

File: preceding-number-25/io.json
Comment (Sandor-B) [inline preceding-number-25/io.json:1]: All good. No mistakes from JSON validator.

File: preceding-number-25/meta.json
Comment (Sandor-B) [inline preceding-number-25/meta.json:1]: All good. No mistakes from JSON validator.

File: preceding-number-25/starter.py
Comment (Sandor-B) [inline preceding-number-25/starter.py:1]: All good. No mistakes.

**Review by Sandor-B — APPROVED** [review]
All files should be good. None of them have any syntax errors.

**Comment by emaj9** [comment]
again good job guys

**Pull request merged**